include/swift/AST/Builtins.def

```diff
@@ -844,6 +844,11 @@ BUILTIN_MISC_OPERATION_WITH_SILGEN(BuildDefaultActorExecutorRef,
 BUILTIN_MISC_OPERATION_WITH_SILGEN(BuildMainActorExecutorRef,
                                    "buildMainActorExecutorRef", "n", Special)
 
+/// isOnStack : (Builtin.NativeObject) -> Builtin.Int1
+///
+/// Returns true if the optimizer successfully moved this object from the stack onto the heap.
```

Comment:
Can you add a comment that this builtin is lowered in the StackSizeLimitTransform?

Reply:
Will do!